### PR TITLE
ineffectual assignment to baseCommand, delete it

### PR DESCRIPTION
**Pull request by yu8833**
Signed-off-by: yupengzte <yu.peng36@zte.com.cn>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by resouer** [comment]
@k8s-bot ok to test

File: cmd/hyperkube/hyperkube.go [hunk 1]
```diff
@@ -144,7 +144,6 @@ func (hk *HyperKube) Run(args []string) error {
 		args = baseFlags.Args()
 		if len(args) > 0 && len(args[0]) > 0 {
 			serverName = args[0]
-			baseCommand = baseCommand + " " + serverName
```
Comment (resouer) [inline cmd/hyperkube/hyperkube.go]: Why this? Could you explain?

Comment (YuPengZTE) [inline cmd/hyperkube/hyperkube.go]: After it, the "baseCommand" is not used.

**Comment by resouer** [comment]
@k8s-bot ok to test

**Comment by mikedanese** [comment]
if this compiles, looks good.

/lgtm

**Pull request merged**